src/ServerScriptService/MainSystem.server.lua

```diff
@@ -4,23 +4,21 @@ local Players = game:GetService("Players")
 -- Example #1: Kicking Players if Datastores are down.
 
 Players.PlayerAdded:Connect(function(player)
-    if DowntimeService:GetDatastoreAPIStatus() == "down" then
-        player:Kick("Datastores are currently down. To prevent critical errors we have kicked you. If this issue persits please contact the game owner. =")
-    end 
+	if DowntimeService.statusFromAlias("datastore")== "Down" then
+		player:Kick("Datastores are currently down. To prevent critical errors we have kicked you. If this issue persits please contact the game owner. =")
+	end 
 end)
 
--- Example #2 Printing latest tests if something is down or not. 
+-- Example #2 Printing the Roblox website's status every 5 minutes. 
 
 while true do
-    
-if DowntimeService:GetSiteStatus() == "down" then
-
-   warn("Roblox Website is Currently Down! Please be patient while Roblox works on a fix! :)") 
-
-elseif DowntimeService.GetSiteStatus() == "degraded" then
-    print("Roblox Website is Currently slow! Please be patient while Roblox works on it! :)")
-elseif DowntimeService:GetSiteStatus() == "up" then
-    print("Roblox website is currently up!")
+	local siteStatus = DowntimeService.statusFromAlias("site")
+	if siteStatus == "Down" then
```

Comment:
It's likely best that you use `string.lower` incase fallen ever decides to pull a fallen and change the text capitalization.